IBPSA/ThermalZones/ReducedOrder/RC/OneElement.mo

```diff
@@ -233,12 +262,29 @@ protected
     "Sums up solar radiation from different directions"
     annotation (Placement(transformation(extent={{-186,118},{-174,130}})));
 
+  Modelica.Blocks.Math.Gain mWat_flow(
+    final k(unit="kg/J") = 1/h_fg,
+    u(final unit="W"),
+    y(final unit="kg/s")) if
+       use_moisture_balance and ATot >0 "Water flow rate due to latent heat gain"
+    annotation (Placement(transformation(extent={{-200,-100},{-180,-80}})));
+
+  Buildings.HeatTransfer.Sources.PrescribedHeatFlow conQLat_flow if
```

Comment:
needs IBPSA counterpart?

Reply:
Yes. Let's let travis fail, then I can add the `after_script` to see if it does the right thing, and then I can fix it.